gnomad_qc/v3/create_release/create_hgdp_tgp_subset.py

```diff
@@ -215,6 +213,13 @@ def prepare_sample_annotations() -> hl.Table:
     )
 
     relatedness_ht = get_relatedness_set_ht(relatedness_ht)
+
+    # Note: Needs to be done before adding the relatedness info because the relatedness HT doesn't have the prefix
+    logger.info(
+        "Removing 'v3.1::' from the sample names, these were added because there are duplicates of some 1KG samples"
+        " in the full gnomAD dataset..."
+    )
+    meta_ht = meta_ht.key_by(s=meta_ht.s.replace("v3.1::", ""))
```

Comment:
I might not have followed the sample complexities, but it seems like there are some tgp samples that are duplicated in the callset mt/meta ht. keying by just "s" here would create two rows for the same sample in the meta ht, right? 

for example, this table:
```
s    info
sample_1    some_info_here
v3.1::sample_1    more_info_here
```
would become this:
```
s    info
sample_1    some_info_here
sample_1    more_info_here
```
would adding `meta_ht = meta_ht.distinct()` help here? (assuming their annotations are all identical)